app/models/user.rb

```diff
@@ -7,7 +7,8 @@ class User < ApplicationRecord
   has_many :assignments
   has_many :messages
 
-  validates :first_name, :last_name, :address, :role, presence: true
```

Comment:
Why are we commenting out these validations and keeping email? This should normally be validated automatically with devise right?